recipes-core/packagegroups/packagefeed-ni-core.bb

```diff
@@ -24,7 +24,6 @@ RDEPENDS:${PN} = "\
 	packagegroup-ni-tzdata \
 	packagegroup-ni-wifi \
 	dkms \
-	bolt \
```

Comment:
I would at least explicitly denote what in systemd obviates bolt and eudev. (My preference would also be to have that in a separate commit, but I will hold that less fast.)